provider/lxd/environ.go

```diff
@@ -133,8 +135,12 @@ func (env *environ) initProfile() error {
 	return err
 }
 
+func (env *environ) shortModelUUID() string {
```

Comment:
This is a duplicate method. The short model uuid is also used an the lxd instance names, thus calculated somewhere already.